src/main/java/com/tstepnik/planner/domain/Task.java

```diff
@@ -23,29 +22,37 @@
 
     private Long userId;
 
-    private ZonedDateTime creationDate;
+    private LocalDateTime creationDate;
 
-    @NotBeforeNow
-    private ZonedDateTime plannedFor;
+    @Future
```

Comment:
Are we sure it should be only future? What about `@FutureOrPresent` 
Btw. I just noticed we don't need to use `LocalDateTime` we only need date :)